package.json

```diff
@@ -51,7 +51,7 @@
     "next": "12.1.4",
     "next-i18next": "10.5.0",
     "next-optimized-images": "3.0.0-canary.10",
-    "next-seo": "5.3.0",
```

Comment:
There is a problem with the newer version? can you add also resolutions field? so it's will be always `5.1.0`

Reply:
I'm just staying aligned with `the-guild-docs`